src/cityreader/cityreader.py

```diff
@@ -17,17 +28,27 @@
 cities = []
 
 def cityreader(cities=[]):
-  # TODO Implement the functionality to read from the 'cities.csv' file
-  # For each city record, create a new City instance and add it to the 
-  # `cities` list
-    
+    with open("cities.csv") as f:
+        csv_reader = csv.reader(f)
+        line = 0
+        for row in csv_reader:
+            if line == 0:
+                line += 1
+            else:
+                name = row[0]
+                lat = row[3]
+                lon = row[4]
+
+                cities.append(City(name, float(lat), float(lon)))
     return cities
 
+
+#  invoking cityreader function and passing in cities array
 cityreader(cities)
 
-# Print the list of cities (name, lat, lon), 1 record per line.
-for c in cities:
```

Comment:
cities = ["Vegas", "Philly", "LA"]
city = cities[0]

for city in cities:
    print(city)